### PR TITLE
KEP-416 Websocket assert fires for read & close operations

### PR DESCRIPTION
**Pull request by ebruck**

### CODE REVIEW

File: node/session.cpp [hunk 1]
```diff
@@ -35,19 +35,7 @@ session::session(std::shared_ptr<bzn::asio::io_context_base> io_context, std::sh
...
-        boost::system::error_code ec;
-
-        this->websocket->get_websocket().close(boost::beast::websocket::close_code::normal, ec);
-
-        if (ec)
-        {
-           LOG(debug) << "failed to close websocket: " << ec.message();
-        }
-    }
+    //LOG(debug) << "shutdown";
```
Comment (amastracci) [inline node/session.cpp]: uncomment or remove

File: node/session.cpp [hunk 2]
```diff
@@ -167,14 +156,13 @@ session::close()
...
+            [self = shared_from_this()](auto ec)
+            {
+                if (ec)
                 {
-                    if (ec)
-                    {
-                        LOG(error) << "failed to close websocket: " << ec.message();
-                    }
-                }));
+                    LOG(error) << "failed to close websocket: " << ec.message();
```
Comment (amastracci) [inline node/session.cpp]: Is this fatal? Should *never* happen, right?

File: node/session.cpp (hunk 1 above)
Comment (ebruck) [reply node/session.cpp]: more for debugging but  I will nuke it

File: node/session.cpp (hunk 2 above)
Comment (ebruck) [reply node/session.cpp]: nope... it's probably already closed

**Pull request merged**